Source/VMware.vSphereDSC/DSCResources/VDPortGroup.ps1

```diff
@@ -287,10 +378,18 @@ class VDPortGroup : BaseDSC {
     #>
     [void] RemoveDistributedPortGroup($distributedPortGroup) {
         try {
-            $distributedPortGroup | Remove-VDPortGroup -Server $this.Connection -Confirm:$false -ErrorAction Stop
+            Write-VerboseLog -Message $this.RemoveVDPortGroupMessage -Arguments @($distributedPortGroup.Name, $distributedPortGroup.VDSwitch.Name)
```

Comment:
**Write-VerboseLog** will need to be replaced by **WriteLogUtil** after the other PR gets approved.